db/migrate/20230329175616_create_like.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.belongs_to :users, null: false, foreign_key: true
```

Comment:
- [ ] From the ERD diagram, likes belongs to both users and posts. Kindly create a new migration to add a foreign key that relates likes and posts please.